### PR TITLE
RBAC for Perspectives and Budgets

### PR DESCRIPTION
**Pull request by charanya-j**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/cloud-cost-management/2-use-cloud-cost-management/8-ccm-dashboards-and-access-control/access-control/manage-access-control-perspective-folders.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	
+10.  To restrict access to specific folders, select **Specified** and then, click **Add**. 
+    
+      ![](./static/add-folders.png)
+
+11. Add the selected folders and save the resource group settings.
+12. You need to bind the **Role** with the **Resource Group**. Go to the **Users** tab on the **Access Control** page.
+13. Select the user you want to restrict access to.
+14. Click **Manage Roles**.
+15. Select the **Role** and **Resource Group** in the **Manage Role Bindings **window.
```
Comment (jyotiarora-harness) [inline docs/cloud-cost-management/2-use-cloud-cost-management/8-ccm-dashboards-and-access-control/access-control/manage-access-control-perspective-folders.md:46]: Please mention here, user have to bind 2 roles, one with the newly created resource group and other one with All resources

Comment (charanya-j) [reply docs/cloud-cost-management/2-use-cloud-cost-management/8-ccm-dashboards-and-access-control/access-control/manage-access-control-perspective-folders.md]: Thank you, Jyoti. I've added this info.

**Comment by charanya-j** [comment]
The feature is yet to be deployed. I'll merge the PR once the feature is deployed.

**Review by iamanmolm — DISMISSED** [review]

File: docs/cloud-cost-management/2-use-cloud-cost-management/8-ccm-dashboards-and-access-control/access-control/manage-access-control-perspective-folders.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    ![](./static/create-new-resource-group.png)
+
+8.  Select **Account only** in the **Resource Scope** dropdown list.
+9.  In the **Resources** pane, select **Cloud Cost management** > **Folders**. 
+    
+      ![](./static/select-resources.png)
+	
+10.  To restrict access to specific folders, select **Specified** and then, click **Add**. 
+
```
Comment (SmritiSatya) [inline docs/cloud-cost-management/2-use-cloud-cost-management/8-ccm-dashboards-and-access-control/access-control/manage-access-control-perspective-folders.md:39]: To restrict access to specific folders, select **Specified** and then click **Add**.

File: docs/cloud-cost-management/2-use-cloud-cost-management/8-ccm-dashboards-and-access-control/access-control/manage-access-control-perspective-folders.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    
+      ![](./static/select-resources.png)
+	
+10.  To restrict access to specific folders, select **Specified** and then, click **Add**. 
+    
+      ![](./static/add-folders.png)
+
+11. Add the selected folders and save the resource group settings.
+12. You need to bind the **Role** with the **Resource Group**. Go to the **Users** tab on the **Access Control** page.
+13. Select the user you want to restrict access to.
```
Comment (SmritiSatya) [inline docs/cloud-cost-management/2-use-cloud-cost-management/8-ccm-dashboards-and-access-control/access-control/manage-access-control-perspective-folders.md:44]: To bind the **Role** with the **Resource Group**, go to the **Users** tab on the **Access Control** page.

File: docs/cloud-cost-management/2-use-cloud-cost-management/8-ccm-dashboards-and-access-control/access-control/manage-access-control-perspective-folders.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+18. Click **Apply**. 
+    
+      ![](./static/manage-role-bindings.png)
+
+
+In Harness, the **Perspectives** page shows only the folders that the user has been granted access to. This means that the user cannot see or interact with any folders that they do not have permission to access.
+
+When it comes to creating Budgets for Perspectives, a user can only do so within the folders that they have been granted access to.
+
+A user with restrictions to folders does not have permission to create new folders. This means that the user can work only within the existing folder structure that has been set up for them by an administrator.
```
Comment (SmritiSatya) [inline docs/cloud-cost-management/2-use-cloud-cost-management/8-ccm-dashboards-and-access-control/access-control/manage-access-control-perspective-folders.md:58]: A user with folder restrictions does not have permission to create new folders. This means that the user can only work within the existing folder structure that has been set up for them by an administrator.

**Review by SmritiSatya — APPROVED** [review]

**Review by SmritiSatya — APPROVED** [review]

**Pull request merged**